PCbuild/pcbuild.proj

```diff
@@ -94,6 +96,11 @@
     <Projects2 Include="venvlauncher.vcxproj;venvwlauncher.vcxproj" />
   </ItemGroup>
 
+  <Target Name="PreBuild" BeforeTargets="Build">
+    <!-- Stick a _PLACEHOLDER=1 after $(PyConfigArgs) to handle both trailing commas and empty $(PyConfigArgs) -->
+    <Exec Command="powershell.exe $(PySourcePath)PCbuild\generate_pyconfig.ps1 -File $(PySourcePath)PC\pyconfig.h -define $(PyConfigArgs)_PLACEHOLDER=1" />
```

Comment:
Would rather have this happen in `pythoncore.vcxproj` if possible, or `_freeze_module.vcxproj` if necessary (or both). That way we don't have to rely on building through this file to make sure it happens.

It probably also belongs in the `regen.targets` file, and can then be referenced from the others.